src/VisualStudio/CSharp/Impl/Options/Formatting/FormattingOptionPageControl.xaml.cs

```diff
@@ -4,7 +4,9 @@
 using Microsoft.CodeAnalysis.Editor.Shared.Options;
 using Microsoft.CodeAnalysis.Formatting;
 using Microsoft.VisualStudio.LanguageServices.Implementation.Options;
+using System.Runtime.CompilerServices;
 
+[assembly: TypeForwardedTo(typeof(CodeStyleNoticeTextBlock))]
```

Comment:
```suggestion

// 🐉 The XAML markup compiler does not recognize InternalsVisibleTo. However, since it allows type
// forwarding, we use TypeForwardedTo to make CodeStyleNoticeTextBlock appear to the markup compiler
// as an internal type in the current assembly instead of an internal type in one of the referenced
// assemblies.
[assembly: TypeForwardedTo(typeof(CodeStyleNoticeTextBlock))]

```

Reply:
added